functions/src/utils.ts

```diff
@@ -11,9 +11,13 @@ const firebaseConfig = process.env.FIREBASE_CONFIG
 // When using `firebase emulators:exec` for testing, firebaseConfig
 // is undefined, so use the dev project as a fallback.
 const firebaseProjectId = firebaseConfig?.projectId ?? "act-now-links-dev";
+// Subdomains of actnowcoalition.org are configured in the Firebase console for each project.
+const subDomain =
```

Comment:
This whole process of deciphering which URL to use is starting to get a little messy 😕 but not sure if there's a simple solution to clean it up...